Add robottelo-ci-update-jobs

elyezer opened the pull request:
The job is going to be responsible to update jenkins every time a new
change is introduced on the master branch.

sthirugn commented on scripts/robottelo-ci-update-jobs.sh:
```diff
@@ -0,0 +1,21 @@
+[jenkins]
+user=${JENKINS_USER}
+password=${JENKINS_PASSWORD}
+url=https://satellite-jenkins.rhev-ci-vms.eng.rdu2.redhat.com
```
can you also move this to internal?

elyezer replied:
My bad

elyezer commented:
@sthirugn I removed your comment to avoid that URL being on history which will continue making public the URL. I have fixed it, thanks for the good catch.

kbidarkar commented:
ACK
